sklearn/gaussian_process/_gpc.py

```diff
@@ -673,6 +673,13 @@ def fit(self, X, y):
         self : object
             Returns an instance of self.
         """
+        if isinstance(
+            self.kernel, CompoundKernel
+        ):  # CompoundKernel is only used internally in scikit-learn
```

Comment:
This comment can be misleading.

```suggestion
        ):
```

It's not "internal" in the sense that is a private api. We do return it here:

https://github.com/scikit-learn/scikit-learn/blob/1d1aadd0711b87d2a11c80aad15df6f8cf156712/sklearn/gaussian_process/_gpc.py#L789-L791

so it is a private API

Reply:
sure, thank you for the advice

Reply:
Sorry, I mistyped here. I meant to say that `CompoundKernel` is a public API.